Helm logo update

flynnduism opened the pull request:
This updates the site header to remove the Microsoft logo at the top of the site - to edit the branding to reflect the community effort.

Updated top logo:

<img width="1355" alt="screen shot 2017-11-14 at 4 21 52 pm" src="https://user-images.githubusercontent.com/686194/32812386-91563306-c959-11e7-8f47-beaf341ae7a1.png">

Contributor logos:

<img width="1300" alt="screen shot 2017-11-14 at 4 22 00 pm" src="https://user-images.githubusercontent.com/686194/32812383-8d265194-c959-11e7-87ee-9b435357c239.png">

**I haven't updated this yet** - wondering who else should be included? Ideally we would give shoutouts to all contributors equally. _Seeking input!_

bacongobbler commented:
We should at the very least add the logos from the core maintainers' companies (with their permission of course! 😄)

From their respective GitHub profiles:

- Nike (@thomastaylor312)
- Intel (@jascott1)
- Mirantis (@nebril)
- IBM (@seh)

And for chart maintainers:

- Ticketmaster (@mgoodness)
- NBC Universal (@scottrigby)
- Samsung SDS (@mattfarina)
- codecentric AG (@unguiculus)

If I missed anyone let me know!

scottrigby commented:
@flynnduism nice work!

@bacongobbler sounds good to me! But don't use the logo on the [NBCUTechnology](https://github.com/NBCUTechnology) github org, as that's the wrong logo (ps that repo is non-representative of the org's public projects, but that's another story). This is the correct logo: https://en.wikipedia.org/wiki/NBCUniversal#/media/File:NBCUNI_Logo.svg

bacongobbler commented on index.html:
```diff
@@ -167,10 +167,32 @@ <h2 class="text-center">Join the Community</h2>
+
+            <div class="row helm-contrib-logos contrib-logos-1">
+              <a target="_blank" href="http://kubernetes.io/"><img src="{{site.baseurl}}/assets/images/logowall/microsoft.png" alt="Microsoft"></a>
+              <a target="_blank" href="http://kubernetes.io/"><img src="{{site.baseurl}}/assets/images/logowall/google.png" alt="Microsoft"></a>
```
looks like the alt tags need updating; they all say "Microsoft".

flynnduism commented:
Thanks @bacongobbler, updated!

flynnduism commented:
Screenshot:

<img width="1288" alt="screen shot 2017-11-15 at 4 04 36 pm" src="https://user-images.githubusercontent.com/686194/32868226-f9b92472-ca25-11e7-843d-d4d26f8c6892.png">

thomastaylor312 commented:
@flynnduism Let's pull out Nike. I work for them, but I don't work on Helm as a Nike employee. Also, I think there is legal stuff involved, which is always...fun. I'll check in to what options we have though for the future. Thanks for putting this together!

bacongobbler commented:
We should just replace Nike's logo with a headshot of @thomastaylor312 ;)

unguiculus commented:
I'm not sure if it makes sense to add logos for individual contributers' companies at all. In any case, I would have to check for codecentric first before we add the logo. Similar to @thomastaylor312 and even though I can do some stuff related to Helm or charts or whatever else during work, I don't officially work on that as codecentric employee.

squillace commented:
I'd love to add those logos for which we have proud approval to put there. but only with approval. Many people contribute formally, and that's fine. But for those who don't, heck, dunno. Certainly no logos without approval. my two cents, anyway.

mgoodness commented:
I'm running this up the flagpole at TM, hoping for approval.

rji commented:
Disclaimer: I am not a lawyer, nor do I represent NIKE, Inc., but I am employed by them.

According to the guidelines at https://help-en-us.nike.com/app/answer/a_id/1071, I do not believe the Nike logo can be used in this context. Further, I would suspect that other organizations have similar logo and trademark restrictions and policies. To echo @squillace's comments, and to avoid any legal ramifications — I would avoid using logos or trademarks without explicit approval.

squillace commented:
What I'd like to end up with is an ongoing list of contributors, and where desired, those entities that support their work, which would include good ol' capitalist mega-entities as well as non-profits and humans doing well by doing good.

mattfarina commented:
I am checking with Samsung. There is an approval request process. For now, don't put it up. I'll let you know if/when there is any kind of approval. Being that this is my first time asking about that kind of thing I have no idea how it's going to go.

bacongobbler commented:
Internally we decided we’re going to remove all logos we don’t have approval at the moment, but we can always add more to the logo wall as @squillace pointed out. That’ll unblock this PR and do what we originally sent out to do: reflect that Helm is a community project; not a Microsoft project. :)

unguiculus commented:
codecentric's logo can be added.
https://github.com/flynnduism/helm-www/pull/1

mgoodness commented:
We're approved to add the TM logo. Updated it in the above PR to @flynnduism's fork/branch.

flynnduism commented:
Thanks for the feedback and logo updates everyone!

I've updated the logo wall and removed any company logos mentioned above that haven't granted permission, and added in a shoutout at the bottom that links to the [contributor list](https://github.com/kubernetes/helm/blob/master/OWNERS).

--

<img width="1197" alt="screen shot 2017-11-20 at 9 55 17 am" src="https://user-images.githubusercontent.com/686194/33033491-5daa422a-cdd9-11e7-8e0e-a49c3d0fe7d0.png">


--

Will do periodic checks to see if anyone wants to add/remove a logo going forward.

bacongobbler commented:
:shipit:
